iroh-dns-server/src/lib.rs

```diff
@@ -101,13 +101,13 @@ mod tests {
                 30,
                 dns::rdata::RData::AAAA(Ipv6Addr::LOCALHOST.into()),
             ));
-            SignedPacket::from_packet(&keypair, &packet)?
+            SignedPacket::new(&keypair, &packet.answers, Timestamp::now())?
```

Comment:
It is not important here, but if there are other places where you have similar verbosity, reminder that the new `SignedPacket::builder()` api is much nicer.